api/v1/User.php

```diff
@@ -45,14 +82,15 @@ public function RequestSession(
         string $country_password = ""
     ): array {
         $response = array();
-        $connection = ConnectionManager::getInstance()->getCachedGameSessionDbConnection($this->getGameSessionId());
+        $connection = ConnectionManager::getInstance()->getCachedServerManagerDbConnection();
         $this->CheckVersion($build_timestamp);
-        $qb = $connection->createQueryBuilder()
-            ->select('game_session_password_admin', 'game_session_password_player')
-            ->from('game_session');
+        $qb = $connection->createQueryBuilder();
+        $qb->select('password_admin', 'password_player')
+            ->from('game_list')
+            ->where($qb->expr()->eq('id', $this->getGameSessionId()));
         $passwords = $connection->executeQuery($qb->getSQL())->fetchAllAssociative();
-        $password_admin = $passwords[0]["game_session_password_admin"];
-        $password_player = $passwords[0]["game_session_password_player"];
+        $password_admin = $passwords[0]["password_admin"];
+        $password_player = $passwords[0]["password_player"];
```

Comment:
This was logging using the session database using :
```
        $connection = ConnectionManager::getInstance()->getCachedGameSessionDbConnection($this->getGameSessionId());
        $this->CheckVersion($build_timestamp);
        $qb = $connection->createQueryBuilder()
            ->select('game_session_password_admin', 'game_session_password_player')
            ->from('game_session');
        $passwords = $connection->executeQuery($qb->getSQL())->fetchAllAssociative();
        $password_admin = $passwords[0]["game_session_password_admin"];
        $password_player = $passwords[0]["game_session_password_player"];
```

And I am still using this way of logging in . So lets check what your idea is switching to the server manager side completely . Before I merge again :)

Reply:
We can merge this after the release 5.1.1